src/main/kotlin/no/nav/syfo/personstatus/infrastructure/kafka/behandlendeenhet/BehandlendeEnhetConsumer.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.syfo.personstatus.infrastructure.kafka.behandlendeenhet
+
+import no.nav.syfo.ApplicationState
+import no.nav.syfo.personstatus.application.PersonBehandlendeEnhetService
+import no.nav.syfo.personstatus.domain.PersonIdent
+import no.nav.syfo.personstatus.infrastructure.kafka.KafkaConsumerService
+import no.nav.syfo.personstatus.infrastructure.kafka.KafkaEnvironment
+import no.nav.syfo.personstatus.infrastructure.kafka.kafkaAivenConsumerConfig
+import no.nav.syfo.personstatus.infrastructure.kafka.launchKafkaTask
+import no.nav.syfo.util.configuredJacksonMapper
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.consumer.ConsumerRecords
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.common.serialization.Deserializer
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.time.OffsetDateTime
+import java.util.Properties
+
+class BehandlendeEnhetConsumer(
+    private val personBehandlendeEnhetService: PersonBehandlendeEnhetService,
+) : KafkaConsumerService<BehandlendeEnhetUpdateRecord> {
+
+    override val pollDurationInMillis: Long = 1000
+
+    override suspend fun pollAndProcessRecords(consumer: KafkaConsumer<String, BehandlendeEnhetUpdateRecord>) {
+        val records = consumer.poll(Duration.ofMillis(pollDurationInMillis))
```

Comment:
Som i de andre consumer'ene er det nok lurt å filtrere bort records med value == null.

Reply:
Gjøres i `processRecords` funksjonen 😊

Reply:
Er du sikker på det? `requireNoNulls()` filtrerer bort de som er null, men ikke records med `value == null`.

Reply:
Aha! Skal dobbelsjekke. Var slik det var satt opp i en annen consumer nemlig, men trenger jo ikke å være riktig for det 😬

Reply:
Hm, da bør vi sikkert ta en gjennomgang på der vi bruker `requireNoNulls`. Det har antagelig ikke skjedd at `record.value()` er `null` fordi da tror jeg vi skulle fått en nullpointer 🤔